x-pack/plugins/apm/public/utils/timepicker/index.js

```diff
@@ -24,6 +24,7 @@ const waitForAngularReady = new Promise(resolve => {
 });
 
 export function initTimepicker(history, dispatch) {
+  return Promise.resolve();
   return new Promise(resolve => {
     // default the timepicker to the last 24 hours
     chrome.getUiSettingsClient().overrideLocalDefault(
```

Comment:
This entire file can be deleted now, right?

Reply:
Yeah, once we're sure it all works.